resources/assets/js/services/utils.js

```diff
@@ -26,4 +26,36 @@ export default {
 
         return (h === '00' ? '' : h + ':') + i + ':' + s;
     },
+
+    /**
```

Comment:
These two would be better pulled in from the Vue core. Wanted to pull in getPath:

https://github.com/vuejs/vue/blob/dev/src/parsers/path.js#L280-L282

So I could better mimic orderBy:

https://github.com/vuejs/vue/blob/dev/src/filters/array-filters.js#L74-L90

But couldn't figure out how.